R/chromote.R

```diff
@@ -603,7 +603,7 @@ is_inside_ci <- function() {
 
 
 is_linux <- function() {
-  Sys.info()[["sysname"]] == "Linux"
+  Sys.info()[["sysname"]] %in% c("Linux", "OpenBSD")
 }
```

Comment:
I think this function can be removed since it's a duplicate of the one in utils.R.

Reply:
Agreed (and it bit me once, because I changed only one instance of it, and it made no difference), but it was not my decision to make. I'll make a separate pull request for this.